src/metabase/email/warn_deprecate_pulse.mustache

```diff
@@ -0,0 +1,16 @@
+{{> metabase/email/_header }}
+<p>Hi {{userName}},</p>
+<p>We deprecated Pulses in Metabase 0.38, two years ago.</p>
+<p>Now the time has come to remove them.</p>
+<p>On the next release, we will delete all Pulses and keep only Dashboard Subscriptions.</p>
+<p>These are the active Pulses you have:</p>
+<ul>
+  {{#pulses}}
+    <li><a href="{{url}}">{{name}}</a></li>
+  {{/pulses}}
+</ul>
+<p>You can transform legacy Pulses into Dashboard Subscriptions following [this doc](https://www.metabase.com/docs/latest/questions/sharing/pulses)</p>
```

Comment:
@jeff-bruemmer, could you help confirm whether this will be the correct doc link?

Reply:
That's the correct link for the latest docs.

I would probably change the language above, though.

> You can transform legacy Pulses into Dashboard Subscriptions following this doc

Transform isn't the right word, as that implies they can take existing pulses and do something to them to turn them into dashboard subscriptions. In reality, they'll need to create new dashboard subscriptions and alerts to replace the pulses.

So maybe:

> You can replace legacy Pulses with [dashboards subscriptions and alerts](https://www.metabase.com/docs/latest/questions/sharing/pulses).

Note also the terminal period, which is missing in the original copy.

Reply:
I think that makes sense, updated it here [0e139d7](https://github.com/metabase/metabase/pull/36392/commits/0e139d730a5e4cbd757e3499e85f53826396b44f) just in case

Reply:
Yeah, works for me